src/zksync-specifics/cheatcodes/zk-use-factory-dep.md

```diff
@@ -0,0 +1,28 @@
+## `zkUseFactoryDep`
+
+### Signature
+
+```solidity
+function zkUseFactoryDep(string calldata name) external pure;
+```
+
+### Description
+
+Enables marking a bytecode as a factory dependency for the next CREATE or CALL and cleared after.
+
+This cheatcode is useful when deploying contracts through factories, as it allows marking bytecode as a factory dependency and deploying contracts by passing their bytecode hash to the factory.
```

Comment:
```suggestion
This cheatcode is useful when deploying contracts through factories that do not depend directly on a given contract, as it allows to explicitly mark this type of contract as a factory dependency, allowing the factory to deploy the contract.

More information on factory dependencies in the [official ZKsync docs](https://docs.zksync.io/build/developer-reference/ethereum-differences/contract-deployment#note-on-factory_deps).
```

Reply:
I like concise but for documentation we should be explicit on what problem this addresses, how it helps, and what you can do with it, outside of the examples later